dice.js

```diff
@@ -0,0 +1,16 @@
+const rollDice = function (){
+    const Dice1 = math.floor(math.random()*(6)+1);
+    const Dice2 = math.floor(math.random()*(6)+1);
+
+    return Dice1+Dice2
+};
+Console.log (rollDice());
+ // BOUNS!
+
+ const rollDice = function (more){
+     let result = null
+     for (let i =more; i>1; i--) {
```

Comment:
a more clean way to do this **for** loop is to set the **start point** `i` to `0` and set  the **end point** to the number of iteration you want `i<=more` and increase the `i` each **step**,
like this :
`for( let i = 0; i <= more; i++)`